projects/plugins/boost/app/assets/src/css/components/card.scss

```diff
@@ -0,0 +1,52 @@
+.jb-card {
+  background: #FFFFFF url("../src/images/color-shift.svg") no-repeat top right;
```

Comment:
We don't include `/src/` in our distributed packages. Images from `/src/` typically get compiled into the JavaScript and embedded that way - but I don't believe CSS will do that. For images included via CSS, use the `assets/static/images` directory.

Reply:
Updated